platform/security/pom.xml

```diff
@@ -152,11 +152,6 @@
                 <version>${jasypt.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.shiro</groupId>
```

Comment:
The same `dependdencyManagement` `dependency` is included in the parent module's pom (`dib-interoperability-app/pom.xml`). Does dependency management work like that?